### ISSUE TITLE
End Logits and Start Logits exactly same values in Windows

### ISSUE THREAD

**Issue by jesus-seijas-sp** [huggingface/node-question-answering#26]
Hi, when running the example in Windows, it returns "Denver" instead of "Denver Broncos".
I tested also in Linux and Mac, in both the result is correct.

I traced the error to this line:
https://github.com/huggingface/node-question-answering/blob/master/src/runtimes/saved-model.worker-thread.ts#L80

The startLogits array and the endLogits array always contains exactly the same values. 
I tested also changing the context and question, and always reproduce the same behaviour: the logits are exactly the same.
